mobify/lockup
# Update deckard - [THIS PR SHOULD MERGE FIRST]

Updating the version of deckard to the unbroken verison :)

Status: **Ready to merge**

Reviewers: @kpeatt 

## Changes
- bumped version of deckard in bower.json

## How to test-drive this PR
- cd into lockup
- `rm -rf bower_components`
- `bower install`
- `grunt test`
- ensure lockup is using the latest version
